_playwright-tests/auth.setup.ts

```diff
@@ -61,6 +63,42 @@ setup.describe('Setup Authentication States', async () => {
     await logout(page);
   });
 
+  setup('Authenticate user with additional subscriptions and save state', async ({ page }) => {
+    setup.setTimeout(60_000);
+
+    // Login layered repo user
+    await logInWithLayeredRepoUser(page);
+
+    // Save state for layered repo user
+    const { cookies } = await page
+      .context()
+      .storageState({ path: path.join(__dirname, '../../.auth', 'layered-repo-user.json') });
+    const layeredRepoToken = cookies.find((cookie) => cookie.name === 'cs_jwt')?.value;
+
+    process.env.LAYERED_REPO_TOKEN = `Bearer ${layeredRepoToken}`;
+
+    await storeStorageStateAndToken(page, 'layered-repo-user.json');
+    await logout(page);
+  });
+
+  setup('Authenticate user with only RHEL subscription and save state', async ({ page }) => {
+    setup.setTimeout(60_000);
+
+    // Login RHEL-only user
+    await logInWithRHELOnlyUser(page);
+
+    // Save state for RHEL-only user
+    const { cookies } = await page
+      .context()
+      .storageState({ path: path.join(__dirname, '../../.auth', 'rhel-only-user.json') });
+    const rhelOnlyToken = cookies.find((cookie) => cookie.name === 'cs_jwt')?.value;
+
+    process.env.RHEL_ONLY_TOKEN = `Bearer ${rhelOnlyToken}`;
+
+    await storeStorageStateAndToken(page, 'rhel-only-user.json');
+    await logout(page);
+  });
+
```

Comment:
both of these auth tests should be skipped unless you are running integration tests, because as it is now you would be required to have the credentials for these very specific users all of the time 🙅🏼 _(which is troublesome for newcomers and it's a bit annoying to need to wait for unnecessary auth steps while testing unrelated stuff)_ 😅 

you can skip these conditionally based on the `INTEGRATION` flag, [here](https://github.com/content-services/content-sources-frontend/pull/601/files#diff-58ff4b40c807fc30b3ab573332f2911976c5a18e353d3d12933f0ea341ec35cfL46) is an example with the `RBAC` one 😅

Reply:
ohhh yep good call, thanks! i'll add that in